nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs

```diff
@@ -96,6 +97,7 @@ protected void Awake()
                 {
                     _grade = filter;
                     SortedData(_grade, _elemental);
+                    AudioController.PlayClick();
```

Comment:
It is recommended to place this line in the input event method.

Reply:
This line already placed in `onValueChanged` event method.